yoti-sdk-impl/src/test/java/com/yoti/api/client/shareurl/policy/SimpleDynamicPolicyBuilderTest.java

```diff
@@ -192,6 +193,48 @@ public void buildWithRememberMeFlags() {
         assertTrue(result.isWantedRememberMeOptional());
     }
 
+    @Test
+    public void buildWithSelfieAuthorisationEnabledThenDisabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withSelfieAuthorisation(true)
+                .withSelfieAuthorisation(false)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), not(hasItem(1)));
+    }
+
+    @Test
+    public void buildWithSelfieAuthorisationDisabledThenEnabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withSelfieAuthorisation(false)
+                .withSelfieAuthorisation(true)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), hasItem(1));
+    }
+
+    @Test
+    public void buildWithPinAuthorisationEnabledThenDisabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withPinAuthorisation(true)
+                .withPinAuthorisation(false)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), not(hasItem(2)));
+    }
+
+    @Test
+    public void buildWithPinAuthorisationDisabledThenEnabled() {
+        DynamicPolicy result = new SimpleDynamicPolicyBuilder()
+                .withPinAuthorisation(false)
+                .withPinAuthorisation(true)
+                .build();
+
+        assertThat(result.getWantedAuthTypes(), hasItem(2));
+    }
+
+
+
```

Comment:
I think there might be too many line breaks here. 
Also please could you add tests to check it handles being initialised with just `.withPinAuthorisation(false)` or `.withPinAuthorisation(false)` 👍

Reply:
Fixed in [022188f]